plugins/org.jboss.tools.openshift.common.ui/plugin.xml

```diff
@@ -258,8 +258,7 @@
 						<iterate
 							ifEmpty="false"
 							operator="and">
-							<instanceof
-								value="org.jboss.tools.openshift.common.core.connection.IConnection" />
+							<adapt	type="org.jboss.tools.openshift.common.core.connection.IConnection" />
```

Comment:
Quote from your linked page:

> Checks if the evaluated object is either an instance of, or adapts to the given class. 
so yes, if the object is an instance of X, <adapt type=x> will evaluate to true